https://issues.redhat.com/browse/ACM-14947 (#7092)
open-cluster-management/rhacm-docs: business_continuity/backup_restore/use_existing_hub_cluster.adoc

[#existing-clusters-detached]
== Existing managed clusters are detached during the restore process

If the hub cluster you plan to use for the restore operation already manages clusters, and you run a restore operation with the `cleanupBeforeRestore` option set to `CleanupRestored`, expect the following outcomes: 

> * If the `ManagedCluster` resources were created by a restored backup, then these resources have the `velero.io/backup-name: backupName` label annotation set. If the user runs a restore operation and uses the `cleanupBeforeRestore=CleanupRestored` option, expect the managed clusters to detach and no longer be part of the current restored backup.

Author: @birsanv of course I want to maintain your original content and its technical accuracy. 

Looking at the Jira, I see the Doc review wants the bullets more user focused and concise. 

Please let me know what you think of this current rewrite, where we say what the earlier users of the managed clusters did, while telling our users/reader of this doc what to expect.

Reviewer: @jc-berger I really like your update ! exactly what I meant

Still one change : 
This : 
expect the managed clusters to detach and no longer be part of the current restored backup.

Should say: 

expect the managed clusters to be detached from the cluster hub and workloads to be cleaned up on these managed clusters.

> * If a user created the `ManagedCluster` resources, then the resources do not have the `velero.io/backup-name: backupName` label annotation set. If you run a restore operation and use the `cleanupBeforeRestore=CleanupRestored` option, the managed cluster resources are not cleaned up even if they are not part of the current restored backup. 

Reviewer 2: ```suggestion
* If a user created the `ManagedCluster` resources, then the resources do not have the `velero.io/backup-name: backupName` label annotation set. If you run a restore operation and use the `cleanupBeforeRestore=CleanupRestored` option, the managed cluster resources are not cleaned up even if the resources are not part of the current restored backup. 
```

> * If the `ManagedCluster` resources were created by a restored backup, then these resources have the `velero.io/backup-name: backupName` label annotation set. If the user runs a restore operation and uses the `cleanupBeforeRestore=CleanupRestored` option, expect the managed clusters to be detached from the cluster hub and workloads to be cleaned up on these managed clusters.
> * If the user created the `ManagedCluster` resources, then these resources do not have the `velero.io/backup-name: backupName` label annotation set. If the user runs a restore operation and uses the `cleanupBeforeRestore=CleanupRestored` option, expect these managed cluster resources to not get cleaned up even if they are not part of the current restored backup. 

Reviewer 2: Are we saying "user" because anyone can run the restore operation? Is there a reason why we should say "user" instead of using "you" (second person according to IBM Style guide)? https://www.ibm.com/docs/en/ibm-style?topic=medium-tone

```suggestion
* If the `ManagedCluster` resources are created by a restored backup, then the resources have the `velero.io/backup-name: backupName` label annotation set. If a user runs a restore operation and uses the `cleanupBeforeRestore=CleanupRestored` option, the managed clusters are detached from the hub cluster and workloads are cleaned up on these managed clusters.
* If a user created the `ManagedCluster` resources, then the resources do not have the `velero.io/backup-name: backupName` label annotation set. If you run a restore operation and use the `cleanupBeforeRestore=CleanupRestored` option, the managed cluster resources are not cleaned up even if they are not part of the current restored backup. 
```

Reviewer: looks good @dockerymick

Reviewer 2: @birsanv is it ok if we say "you" instead of "a/the user"?

Reviewer: @dockerymick 
for this I think we should stick with `a user` ( because it can be any other user , not the one trying to run the restore operation )
If **a user** created the `ManagedCluster` resources, then the resources do not have the `velero.io/backup-name: backupName` label annotation set.

For this I think we can use 'you' since it's about the current operation
If a user runs a restore... ( If **you** run a restore .. )
